index.js

```diff
@@ -0,0 +1,46 @@
+const { resolve } = require("path");
+const lib = require("./lib.js");
+
+var completeData = lib.completeData
```

Comment:
Where is it used?

Reply:
sir, i didn't added require statement when i wrote promise the text editor added to it